pkg/limits/config.go

```diff
@@ -77,6 +81,12 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 		DefaultBucketSize,
 		"The size of the buckets used to calculate stream rates. Smaller buckets provide more precise rates but require more memory.",
 	)
+	f.DurationVar(
+		&cfg.EvictInterval,
+		"ingest-limits.evict-interval",
```

Comment:
Can we call this `eviction` instead of `evict`? I looked at other Loki flags and we use the word `eviction`. It just feels nice and consistent 😄 

```suggestion
		"ingest-limits.eviction-interval",
```